ts/src/bitfinex2.ts

```diff
@@ -2460,8 +2461,77 @@ export default class bitfinex2 extends Exchange {
         //         ]
         //     ]
         //
-        // todo unify parsePosition/parsePositions
-        return response;
+        return this.parsePositions (response);
+    }
+
+    parsePosition (position, market = undefined) {
+        //
+        //    [
+        //        "tBTCUSD",                    // SYMBOL
+        //        "ACTIVE",                     // STATUS
+        //        0.0195,                       // AMOUNT
+        //        8565.0267019,                 // BASE_PRICE
+        //        0,                            // MARGIN_FUNDING
+        //        0,                            // MARGIN_FUNDING_TYPE
+        //        -0.33455568705000516,         // PL
+        //        -0.0003117550117425625,       // PL_PERC
+        //        7045.876419249083,            // PRICE_LIQ
+        //        3.0673001895895604,           // LEVERAGE
+        //        null,                         // _PLACEHOLDER
+        //        142355652,                    // POSITION_ID
+        //        1574002216000,                // MTS_CREATE
+        //        1574002216000,                // MTS_UPDATE
+        //        null,                         // _PLACEHOLDER
+        //        0,                            // TYPE
+        //        null,                         // _PLACEHOLDER
+        //        0,                            // COLLATERAL
+        //        0,                            // COLLATERAL_MIN
+        //        // META
+        //        {
+        //            "reason": "TRADE",
+        //            "order_id": 34271018124,
+        //            "liq_stage": null,
+        //            "trade_price": "8565.0267019",
+        //            "trade_amount": "0.0195",
+        //            "order_id_oppo": 34277498022
+        //        }
+        //    ]
+        //
+        const marketId = this.safeString (position, 0);
+        const amount = this.safeString (position, 2);
+        const timestamp = this.safeString (position, 12);
+        const meta = this.safeString (position, 19);
+        const tradePrice = this.safeString (meta, 'trade_price');
+        const tradeAmount = this.safeString (meta, 'trade_amount');
+        return this.safePosition ({
+            'info': position,
+            'id': this.safeString (position, 11),
+            'symbol': this.safeSymbol (marketId, market),
+            'notional': amount,
+            'marginMode': 'isolated',  // derivatives use isolated, margin uses cross, https://support.bitfinex.com/hc/en-us/articles/360035475374-Derivatives-Trading-on-Bitfinex
+            'liquidationPrice': this.safeString (position, 8),
```

Comment:
@samgermain right now `safePosition` does not convert the strings to `numbers`, so we have to use safeNumber